### ISSUE TITLE
auto completion works well on Linux but not on Windows

### ISSUE THREAD

**Issue by jdhao** [srusskih/SublimeJEDI#242]
I have installed SublimeJEDI with Sublime Text 3 both on Windows and CentOS. On both platform, I use the python shipped with Anaconda. But I find that it does not work well on Windows. I have set it to complete on dot in my preference settings,

```
{
    // ...
    "auto_complete_triggers": [{"selector": "source.python", "characters": "."}],
}
```

In order to debug, I use the setting

```
{
     "logging_level": "info",
}
```

in the JEDI setting file. I have observed that JEDI immediately start completing after I type triggers such as `.`, `i` on CentOS but not on Windows. 

How to deal with this issue?

**Comment by srusskih** [comment]
does it completion on TAB or CTRL+SPACE (or another hot keys that triggers completion) ?

**Comment by jdhao** [comment]
On windows, in order to auto-complete, for the `import` command, I have type import and then type at least the initial character of the package, then if I press TAB, completion will work. For the method completion to work, I have to type `.`, then at least type the first character of the method and press TAB to trigger completion. 

This behavior is different from the Linux platform and is awkward.

**Comment by srusskih** [comment]
does sublime text user preferences same for both systems ?

**Comment by jdhao** [comment]
Yes, the settings are the same.  In the `sublime_jedi.sublime-settings` file, I use the following options:

```
{
    "auto_complete_function_params": "",
    "logging_level": "info",
    "sublime_completions_visibility": "default"
}
```

In the sublime preference settings, I add the following auto-complete triggers,

	"auto_complete_triggers":
	[
		{
			"characters": ".",
			"selector": "source.python"
		}
	],

Also, according to [trouble shooting](https://github.com/srusskih/SublimeJEDI#troubleshooting), I removed `import` from the file `Completion Rules.tmPreferences`.

**Comment by srusskih** [comment]
will try to find windows somewhere to check. currently have no clue what the reason.

**Comment by jdhao** [comment]
In order to better illustrate the idea. I have made two images

+ Windows behaviour
![windows_behavior](https://user-images.githubusercontent.com/16662357/39711965-fac16a26-5253-11e8-8fc5-f0e560f178de.gif)

+ Linux behaviour
![linux_behavior](https://user-images.githubusercontent.com/16662357/39712022-21a2efd4-5254-11e8-8e08-9821aed87bfe.gif)

From the two image, we can see that on Windows, the auto-completion is kind of slow and "lazy", it will work only after I have type `.`  and initial character of a method and press TAB.

But on Linux, the auto-completion starts to work once I begin to type. Also for methods completion, once I have type `.`, it will start to complete.  No need to press TAB.

The behaviors on these two system are just way too different.
